tests/lean/inductionErrors.lean

```diff
@@ -79,3 +79,9 @@ theorem ex11 (p q : Nat) : p ≤ q ∨ p > q := by
   | upper d => apply Or.inr; admit
   | lower d /- error duplicate -/ => apply Or.inl; admit
   | diag    => apply Or.inl; apply Nat.le_refl
+
+theorem ex12 (p: Nat) : p ≤ q ∨ p > q := by
+  cases p, q using elimEx with
+  | lower2 /- error -/ d => apply Or.inl; admit
+  | upper d => apply Or.inr
+  | diag    => apply Or.inl; apply Nat.le_refl
```

Comment:
These tests with the `.expected.out` output are a bit out of fashion. Can you add a new test file in the `tests/lean/run` directory and use `#guard_msgs in` to assert the expected output? Also makes it easier to review.